src/components/SearchAndFilter/SearchAndFilter.js

```diff
@@ -24,9 +20,6 @@ const SearchAndFilter = ({
 
```

Comment:
Not in the diff, but if you could fix it please, `useRef` should never be called without an initial value. In both cases, you probably want `useRef(null)` here.